data-collection/deploy/module-inventory.yaml

```diff
@@ -878,6 +915,36 @@ Resources:
               except Exception as exc:  #pylint: disable=broad-exception-caught
                   logger.info(f'scan {service}/{account_id}/{region}: {exc}')
 
+          def eks_clusters_scan(account_id, region):
+            """special function to scan EKS clusters"""
+            service = "eks"
+            session = assume_session(account_id, region)
+            client = session.client(service, region_name=region)
+            try:
+                paginator = client.get_paginator("list_clusters")
+                page_iterator = paginator.paginate(
+                    PaginationConfig={
+                        "PageSize": 100,
+                    },
+                )
+                clusters_names = []
+                for page in page_iterator:
+                    clusters_names.extend(page["clusters"])
```

Comment:
can be 

`for cluster_name in client.get_paginator("list_clusters").paginate().search('clusters'):`